include/NovelRT.Interop/Audio/NrtAudioService.h

```diff
@@ -11,32 +11,38 @@ extern "C"
 {
 #endif
 
-    NrtAudioServiceHandle Nrt_AudioService_create();
-    NrtResult Nrt_AudioService_destroy(NrtAudioServiceHandle service);
+    NrtAudioServiceHandle Nrt_AudioService_Create();
+    NrtResult Nrt_AudioService_Destroy(NrtAudioServiceHandle service);
 
-    NrtBool Nrt_AudioService_initialiseAudio(NrtAudioServiceHandle service);
-    NrtResult Nrt_AudioService_loadMusic(NrtAudioServiceHandle service,
+    NrtBool Nrt_AudioService_InitialiseAudio(NrtAudioServiceHandle service);
+    NrtResult Nrt_AudioService_LoadMusic(NrtAudioServiceHandle service,
                                          char* input,
                                          NrtAudioServiceIteratorHandle* output);
 
-    NrtResult Nrt_AudioService_setSoundVolume(NrtAudioServiceHandle service, unsigned int source, float val);
-    NrtResult Nrt_AudioService_setSoundPosition(NrtAudioServiceHandle service,
+    NrtResult Nrt_AudioService_SetSoundVolume(NrtAudioServiceHandle service, unsigned int source, float val);
+    NrtResult Nrt_AudioService_SetSoundPosition(NrtAudioServiceHandle service,
                                                 unsigned int source,
                                                 float posX,
                                                 float posY);
-    NrtResult Nrt_AudioService_resumeMusic(NrtAudioServiceHandle service);
-    NrtResult Nrt_AudioService_playMusic(NrtAudioServiceHandle service,
+    NrtResult Nrt_AudioService_ResumeMusic(NrtAudioServiceHandle service);
+    NrtResult Nrt_AudioService_PlayMusic(NrtAudioServiceHandle service,
                                          NrtAudioServiceIteratorHandle handle,
                                          int32_t loops);
```

Comment:
mega nit: Inconsistent spacing between the signatures here. I'm guessing clang-format did this? Looks weird either way. Don't spend too long on it if clang-format refuses to cooperate though.

Reply:
Yup, this was 100% clang-format as we've set it up to do this, however are you referring to the parameter spacing here?

Reply:
No, I'm referring to the fact that multi-line signatures have no spacing between each other, like:
```cpp
MyThing(A a,
        B b);
MyOtherThing(A a,
             B b);
```

As opposed to:
```cpp
MyThing(A a,
        B b);
        
MyOtherThing(A a,
             B b);
```

Reply:
Not sure - could've been a side effect that wasn't manually resolved.

Cleaned it up now